bot/exts/evergreen/catify.py

```diff
@@ -0,0 +1,44 @@
+import random
+from typing import Optional
+
+from discord.ext import commands
+
+from bot.constants import Cats
+
+
+class Catify(commands.Cog):
+    """Cog for the catify command."""
+
+    def __init__(self, bot: commands.Bot):
+        self.bot = bot
+
+    @commands.command(aliases=["ᓚᘏᗢify", "ᓚᘏᗢ"])
+    async def catify(self, ctx: commands.Context, *string: Optional[str]) -> None:
```

Comment:
I'd prefer this argument to be called `text` for better UX, string is too technical.

Reply:
did this